ganga/GangaGUI/start.py

```diff
@@ -0,0 +1,46 @@
+from GangaCore.Core.GangaThread import GangaThread
+from GangaCore.Utility.logging import getLogger
+from GangaGUI.gui import app
+import requests
+
+logger = getLogger()
+gui_server = None
+
+
+# GangaThread for Flask Server
+class GUIServerThread(GangaThread):
+
+    def __init__(self, name: str, host, port):
+        GangaThread.__init__(self, name=name)
+        self.host = host
+        self.port = port
+
+    def run(self):
+        try:
+            logger.info("Starting GUI Server at {}:{}".format(self.host, self.port))
+            app.run(host=self.host, port=self.port)
+        except Exception as err:
+            logger.error("Failed to start GUI Server: {}".format(err))
+
+    def shutdown(self):
+        res = requests.post("http://localhost:{port}/shutdown".format(port=self.port))
+        logger.info("{} - {}".format(res.status_code, res.text))
+
+
+def start_gui(host: str = "localhost", port: int = 5000):
+    """Start GUI Flask App on a GangaThread"""
```

Comment:
As these functions will be exported to the user, it is important that the doc lines explain well what is going on. For now it is fine though.

Reply:
Okay, I will add the usage section also in the docstring along with more details on how it runs.